ras/ras_lsvpd.py

```diff
@@ -76,7 +76,8 @@ def test_build_upstream(self):
                 deps.extend(['libsgutils-devel', 'sqlite3-devel',
                              'libvpd2-devel'])
             elif self.detected_distro.name in ['centos', 'fedora', 'rhel']:
-                deps.extend(['sqlite-devel', 'libvpd-devel'])
+                deps.extend(['sqlite-devel', 'libvpd-devel',
+                             'sg3_utils-devel'])
```

Comment:
@sacsant is this from specific distro version?
in that case we have to handle to differently.

if it is for across the distro versions of `['centos', 'fedora', 'rhel']` then is this missed earlier?

Reply:
This is applicable for all the variants. I missed it the previous patch as this particular package was already installed. I ran into this error when I attempted to run the test on a freshly installed system.